src/main/java/frc/robot/Constants.java

```diff
@@ -29,10 +29,9 @@
   public static final int SPARKMAX_SHOOTER_CAN_ADDR_RIGHT = 5;
   public static final int SPARKMAX_SHOOTER_CAN_ADDR_LEFT = 6;
   public static final int SPARKMAX_LOADER_CAN_ADDR = 7;
```

Comment:
What is "Loader"

Reply:
I think I put that there while working on the shooter.  It's a deprecated version of the shooter gate, which is under StorageSubsystem